src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -279,12 +279,17 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
     // Note that the date/time helpers used here were registered in the home templates
     function tourDestinationList(tour) {
         var source = [
-        '<ul class="tour-list">',
+        '<div class="tour-list">',
             '<div class="tour-heading">',
                 '<div class="tour-label">',
                     '{{#if tour.is_event}}Event{{else}}Tour{{/if}}',
                 '</div>',
-                '<h1 class="tour-name">{{ tourName }}</h1>',
+                '<h1 class="tour-name">',
+                    '<a class="tour-name-link" href=',
+                        '"/{{#if tour.is_event}}event{{else}}tour{{/if}}/{{ tour.id }}/">',
```

Comment:
This should probably `target="_blank"` to open in a new tab, as the other links do.

Reply:
The app generally only opens _external_ pages in new tabs, whereas this link is for navigating within the app, so I'm going to leave as is. (I poked around the code to verify this and found a `target="blank"` that breaks this logic, so I've pushed a fix for that.)